tools/third-party-dependencies/known-dependencies.txt

```diff
@@ -124,4 +124,10 @@ system-rules-1.16.1.jar
 truth-0.30.jar
 zipkin-2.23.2.jar
 zipkin-reporter-2.16.3.jar
-zipkin-sender-okhttp3-2.16.3.jar
\ No newline at end of file
+zipkin-sender-okhttp3-2.16.3.jar
+httpasyncclient-4.1.3.jar
+httpcore-nio-4.4.6.jar
+javassist-3.21.0-GA.jar
+nacos-client-2.0.4.jar
+reflections-0.9.11.jar
+snakeyaml-1.23.jar
```

Comment:
What is a NOTICE？

Reply:
According to [Apache license](https://www.apache.org/licenses/LICENSE-2.0.html#redistribution), We need to add notice of third-party jar which is in Apache license into our NOTICE file.  But we are missing the NOTICE file :). @xwm1992 This might be an urgent thing need to fix.

Reply:
> According to [Apache license](https://www.apache.org/licenses/LICENSE-2.0.html#redistribution), We need to add notice of third-party jar which is in Apache license into our NOTICE file. But we are missing the NOTICE file :). @xwm1992 This might be an urgent thing need to fix.

I think there may be a lot to add, you can separate the issue to operate